src/LibObjectFile/Elf/ElfReader{TDecoder}.cs

```diff
@@ -508,7 +508,7 @@ private void VerifyAndFixProgramHeadersAndSections()
             if (hasShadowSections)
             {
                 int shadowCount = 0;
-                uint previousSectionIndex = 1;
+                uint previousSectionIndex = 0;
```

Comment:
I will have to double check, but it was 1 originally for the index 0 which is supposed to be the null section.